.travis.yml

```diff
@@ -83,15 +83,12 @@ sudo: required
 
 script: $PIWIK_ROOT_DIR/tests/travis/travis.sh
 
-before_install:
-  # do not use the Zend allocator on PHP 5.3 since it will randomly segfault after program execution
-  - '[[ "$TRAVIS_PHP_VERSION" == 5.3* ]] && export USE_ZEND_ALLOC=0 || true'
-
 install:
   - git fetch -q
 
-  - export GENERATE_TRAVIS_YML_COMMAND="php ./tests/travis/generator/main.php generate:travis-yml --core --verbose"
-  - '[[ "$TRAVIS_JOB_NUMBER" != *.1 || "$TRAVIS_PULL_REQUEST" != "false" ]] || ./tests/travis/autoupdate_travis_yml.sh'
+  # Disable it until this is in master, otherwise we have to create a branch for travis submodule
```

Comment:
I really wanted to avoid having to keep a branch of travis submodule in sync. I reckon it's ok to just not have this activated for now. Not sure how to make sure we won't forget to enable it again. Of course we can create an issue but chances are high we won't look into that issue... I will create one though if we merge this PR